be/src/olap/rowset/segment_v2/column_writer.h

```diff
@@ -38,6 +39,7 @@ struct ColumnWriterOptions {
     CompressionTypePB compression_type = NO_COMPRESSION;
     bool need_checksum = false;
     size_t data_page_size = 64 * 1024;
+    double min_compress_ratio = 0.9;
```

Comment:
the name is confusing because compression ratio is conventionally calculated as `uncompressed_size / compressed_size`. I think `space saving` is more appropriate here, see the definition in https://en.wikipedia.org/wiki/Data_compression_ratio 

```suggestion
    // store compressed page only when space saving is above the threshold.
    // space saving = 1 - compressed_size / uncompressed_size
    double compression_min_space_saving = 0.1;
```

Reply:
OK, I will change it